<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StateSetView</class>
 <widget class="QWidget" name="StateSetView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>基本</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>名称：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="name">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>纹理</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTreeView" name="texture_tree_view"/>
       </item>
       <item>
        <widget class="QStackedWidget" name="texture_stacked">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="texture">
          <layout class="QGridLayout" name="gridLayout" columnstretch="0,1">
           <item row="6" column="1">
            <widget class="QComboBox" name="min_filter_combo_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR_MIPMAP_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR_MIPMAP_NEAREST</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST_MIPMAP_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST_MIPMAP_NEAREST</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>名称</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="texture_file_label">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>MAG_FILTER</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="wrap_r_combo_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_EDGE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_BORDER</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>REPEAT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MIRROR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>WRAP_S</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>WRAP_T</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="wrap_s_combo_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_EDGE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_BORDER</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>REPEAT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MIRROR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>MIN_FILTER</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>文件</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="wrap_t_combo_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_EDGE</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CLAMP_TO_BORDER</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>REPEAT</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MIRROR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>WRAP_R</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="texture_name_label">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QComboBox" name="mag_filter_combo_box">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR_MIPMAP_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LINEAR_MIPMAP_NEAREST</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST_MIPMAP_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NEAREST_MIPMAP_NEAREST</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="texgen">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Mode</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="mode_combo_box">
             <item>
              <property name="text">
               <string/>
              </property>
             </item>
             <item>
              <property name="text">
               <string>OBJECT_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>EYE_LINEAR</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SPHERE_MAP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>NORMAL_MAP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>REFLECTION_MAP</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="material_tab">
      <attribute name="title">
       <string>材质</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1">
       <item row="4" column="1">
        <widget class="QWidget" name="shininess_front_spin_box" native="true"/>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="ambient_front_spin_box" native="true"/>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="diffuse_front_spin_box" native="true"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Specular(FRONT)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Ambient(FRONT)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Emission(FRONT)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Diffuse(FRONT)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Shininess(FRONT)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="emission_front_spin_box" native="true"/>
       </item>
       <item row="3" column="1">
        <widget class="QWidget" name="specular_front_spin_box" native="true"/>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
